hs_core/search_indexes.py

```diff
@@ -184,8 +184,10 @@ def prepare_author_raw(self, obj):
         """
         if hasattr(obj, 'metadata'):
             first_creator = obj.metadata.creators.filter(order=1).first()
-            if first_creator.name is not None:
+            if first_creator.name:
                 return first_creator.name.lstrip()
+            elif first_creator.organization:
+                return first_creator.organization.strip()
```

Comment:
If stripping organization is a concern... any reason why we aren't doing this on save rather than on view?

Reply:
@sblack-usu Not quite sure about your comment here. This is just prepare first author solr field for solr search using resource metadata creators model. strip() call is just to strip potential white spaces for better presentation on discovery page. Can you elaborate a bit on your comment since I don't get it?

Reply:
Right, I'm just suggesting it might be better to normalize (strip whitespace) when the author is created or updated.  Just so we have clean data in the database that is ready to be displayed, rather than stripping the whitespace just before displaying (there could be many places we're displaying and we may or may not be stripping whitespace on display).  Just a thought and I think it's not needed for this ticket.

Reply:
@sblack-usu Got it. That makes total sense to me. I think most likely we are already stripping out whitespaces in organization when the author is initially created. I am doing strip() call here just as a precaution since I see strip() call has been done for creator.name in this same routine.